src/utils/plotSpecs/generateVolcanoSpec.js

```diff
@@ -1,28 +1,8 @@
 import _ from 'lodash';
 
-const generateSpec = (configSrc, data) => {
+const generateSpec = (configSrc, plotData) => {
   const config = _.cloneDeep(configSrc);
 
-  let maxNegativeLogpValue = 0;
-
-  data.forEach((o) => {
-    Object.keys(o).forEach((k) => {
-      if (k === 'p_val_adj' && o[k] && o[k] !== 0) {
-        maxNegativeLogpValue = Math.max(
-          maxNegativeLogpValue, -Math.log10(o[k]),
-        );
-      }
-    });
-  });
-
-  const logFoldChangeFilterExpr = (config.logFoldChangeDomain)
-    ? `datum.logFC > ${config.logFoldChangeDomain * -1} && datum.logFC < ${config.logFoldChangeDomain}`
-    : 'true';
-
-  const negativeLogpValueFilterExpr = (config.maxNegativeLogpValueDomain)
-    ? `datum.neglogpvalue < ${config.maxNegativeLogpValueDomain}`
-    : 'true';
-
```

Comment:
This is removed because we do not want to filter out user's data that lies out of the plot area